src/command/feed/print.ts

```diff
@@ -27,7 +27,7 @@ export class Print extends FeedCommand implements LeafCommand {
     const addressString = this.address || (await this.getAddressString())
     const reader = this.bee.makeFeedReader('sequence', topic, addressString)
     const { reference, feedIndex, feedIndexNext } = await reader.download()
-    const manifest = await this.bee.createFeedManifest('sequence', topic, addressString)
+    const manifest = await this.bee.createFeedManifest(this.stamp, 'sequence', topic, addressString)
```

Comment:
Read operation should not use stamp

Reply:
I agree that printing should not require a stamp, however currently we cannot calculate the feed manifest address on the client side so we rely on calling the `createFeedManifest` which uploads the feed manifest.

Until we implement the feed manifest calculation on the client side, I see the following options:
1. We don't print the manifest address. This is not great :)
2. We always require the stamp for printing the feed. This may be surprising to the users but it will work and print the manifest address.
3. If a stamp is provided as an optional argument we print the feed manifest address, otherwise not. This can be confusing to explain, but the default behavior will be the way it is supposed to be used in the future.

What do you think?

Reply:
I think this is not a blocker, and we should keep it in its current state. Make it require a stamp so it can print the manifest. That is probably the most important part of the `feed print` command.

Then, as part of the next steps, let's implement this the right way with a client side calculation.